docker.sh

```diff
@@ -33,12 +31,11 @@ COMMIT=$(git rev-parse HEAD)
 echo "COMMIT=\"${COMMIT}\"" > indexd/index/version_data.py
 
 BUILD_COMMAND=(build \
-  --label org.opencontainers.image.version="${VERSION}" \
   --label org.opencontainers.image.created="$(date -Iseconds)" \
   --label org.opencontainers.image.revision="${COMMIT}" \
-  --label org.opencontainers.ref.name="${NAME}:${GIT_BRANCH}" \
+  --label org.opencontainers.image.ref.name="${NAME}:${GIT_BRANCH}" \
```

Comment:
Maybe skip this for now?
Or keep both?
(otherwise, this will be correct while all others aren't)

Reply:
added